cloud-native-runtimes/how-to-guides/auto-tls/tls-guides-deactivate-autotls.hbs.md

```diff
@@ -1,6 +1,6 @@
 # Opt out from an ingress issuer and deactivate automatic TLS feature
 
-This topic tells you how to opt out from an ingress issuer and deactivate automatic TLS feature for Cloud Native Runtimes, commonly known as CNR.
+This topic tells you how to opt out from an ingress issuer and deactivate automatic TLS feature for Cloud Native Runtimes.
```

Comment:
Why here we remove the "commonly known as..." but on verifying_services we keep it?

Reply:
Good point. I wanted to remove it because it seems repetitive but its not a big deal. I will put it back